src/sentry/integrations/github/webhook.py

```diff
@@ -47,6 +47,18 @@ def get_github_external_id(event: Mapping[str, Any], host: str | None = None) ->
     return f"{host}:{external_id}" if host else external_id
 
 
+def get_file_language(filename: str):
```

Comment:
imagine not typing lul
```suggestion
def get_file_language(filename: str) -> Union[str, None]:
```

Reply:
typing check didn't complain 👀

Reply:
on new files our typing is not set to strict -- but nice stuff. I think `str | None` is slightly preferred.